examples/waspleau/README.md

```diff
@@ -0,0 +1,25 @@
+# Waspleau
+
+Welcome to the Waspleau example! This is a small Wasp project that will allow you to setup an easy Dashboard that pulls and stores data in-memory using scheduled jobs on a Redis-backed Bull queue.
+
+Note: The commands below assume we are using PostgreSQL, but this is only due to a known bug that requires a model to use queries (even though we don't use it here). We are tracking it here: https://github.com/wasp-lang/wasp/issues/398
+
+## Step 1
+
+Clone this repo
+
+## Step 2
+
+Update ext/workers with whatever you want to track and import in `serverSetup.js`
+
+## Step 3 (with PostgreSQL running)
+
+`DATABASE_URL="postgresql://postgres@localhost/waspleau-dev" wasp db migrate-dev`
+
+## Step 4
+
+`DATABASE_URL="postgresql://postgres@localhost/waspleau-dev" wasp start`
+
+This will start your background workers and present a dashboard UI that will auto-refresh every minute.
+
+Note: As you develop your own workers, keep in mind each time you save a file in the project it will automatically reload, hence restarting your server and server function. Use the `MOCK_DATA=true` command line envar if you want to avoid pinging external resources.
```

Comment:
I guess what would help here is if we had hot reloading? I am not sure if it can be applied to this, but if it could, it would be cool.

Reply:
Yeah I haven't looked into it much tbh but it would be nice to toggle in this use case.